pre-commit/web_lint_check.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+D=webui/react
+target=$1
+files=$(realpath --relative-to="$D" "${@: 2}" | tr "\n" " ")
```

Comment:
The arg array slice is a Bash extension.  It'd be more portable to just pop the first arg off with shift.
```suggestion
shift
files=$(realpath --relative-to="$D" "$@" | tr "\n" " ")
```
That's important, since the bang path and command run in precommit is `sh` rather than `bash`.  In general, it's not safe to assume that /bin/sh is bash; in particular, /bin/sh is `dash` on Debian-derived systems.

Reply:
I fixed it!